linux/net/rina/ipcps/Makefile

```diff
@@ -10,5 +10,7 @@ ccflags-y += -Wtype-limits
 
 obj-$(CONFIG_RINA_NORMAL_IPCP)    += normal-ipcp.o
 obj-$(CONFIG_RINA_SHIM_ETH_VLAN)  += shim-eth-vlan.o
+shim-eth-vlan-y :=				\
```

Comment:
No, I've checked https://www.kernel.org/doc/Documentation/kbuild/makefiles.txt, sections 3.2 and 3.3. If the shim-eth-vlan is not enabled for compilation the makefile will be

obj-n += shim-eth-vlan.o 

and the module will be ignored. The second line is just in case the module has multiple files, and it always needs to be <module name>-y.

I tried with the replacement and produced an error when compiling (that's why I checked the syntax).